jobs/client-regeneration/client_regeneration/main.py

```diff
@@ -0,0 +1,383 @@
+import click
+
+from google.cloud import bigquery
+from datetime import datetime, timedelta
+
+DEFAULT_START_DATE = "2022-04-01"
+DEFAULT_END_DATE = "2022-06-01"
+
+COLUMN_LIST = [
```

Comment:
let's just use these columns for now. 

```suggestion
COLUMN_LIST = [
   "country",
    "device_model",
    "device_manufacturer",
]
```